helm/scylla/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if (and .Values.serviceAccount.create .Values.serviceAccount.annotations) -}}
+# Even though Scylla Operator creates and manages Service Account used by ScyllaClusters
+# we have to keep it here to keep backward compatibility of setting custom annotations via helm chart.
+# Operator will adopt it anyways on creation and updates.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "scylla.serviceAccountName" . }}
```

Comment:
does this need to be fixed?

Reply:
it was defined in _helpers. Moved it here.